### DISCUSSION TITLE
Support for Isaac Sim 4.0.0

### DISCUSSION THREAD

**Discussion by sjauhri** [NVlabs/curobo#293] [Software Q&A]
Hello team,

I wanted to ask when curobo is expected to support the newest Isaac Sim version 4.0.0 (https://forums.developer.nvidia.com/t/isaac-sim-4-0-availability-announced-at-computex-2024/294912)

If support is expected soon, I would switch to the latest version, otherwise not

**Comment by balakumar-s** [comment]
We are still working through the URDF importer part of cuRobo (at least not until end of July). In the mean time, if you already have your robot loaded in Isaac sim 4.0 , you can connect the robot through the python API.

**Reply by shengyangzhuang** [reply]
Is there a way to download the isaac sim 2022.2.1 workstation now? The launcher only allows us to download 4.0.0 or 2023.1.1. I couldn't build the docker of curobo with Isaac sim 2022.2.1. I can only build 2023.1.0 one but most examples couldn't work. I know we can still get 2022 version docker with NGC, but it wont be built with curobo.
